doc/releases/changelog-0.21.0.md

```diff
@@ -315,11 +310,32 @@
 
 <h3>Improvements</h3>
```

Comment:
For the improvements section, I would move the following to the top:

- The insert transform now supports adding operation after or before certain specific gates. [(#1980)](https://github.com/PennyLaneAI/pennylane/pull/1980)

- Added a modified version of the simplify function to the hf module. [(#2103)](https://github.com/PennyLaneAI/pennylane/pull/2103)

- The QAOA module now accepts both NetworkX and RetworkX graphs as function inputs. [(#1791)](https://github.com/PennyLaneAI/pennylane/pull/1791)

Reply:
Thanks!